src/frontends/tensorflow/src/translate_session.cpp

```diff
@@ -66,7 +66,16 @@ void TranslateSession::inject_body_model(std::shared_ptr<ov::Model> body_model,
                             "inputs and arguments to the function " +
                                 operation_type + " do not match.");
     for (size_t param_ind = 0; param_ind < body_parameters.size(); ++param_ind) {
+        auto orig_type = body_parameters[param_ind]->get_element_type();
         body_parameters[param_ind]->output(0).replace(ov_inputs[param_ind]);
+        if (auto ext_parameter = as_type_ptr<ov::opset8::Parameter>(ov_inputs[param_ind].get_node_shared_ptr())) {
```

Comment:
Yeah, this is annoying thing from which I want to get rid of in the separate PR. Actually, I am afraid to use opset10 for Parameter because everywhere in TF FE we use opset8 for Parameter.